src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -130,6 +136,13 @@ public void InitializeIsExtensionMethod(bool isExtensionMethod)
                 ThreadSafeFlagOperations.Set(ref _bits, bitsToSet);
             }
 
+            public void InitializeIsReadOnly(bool isReadOnly)
```

Comment:
>InitializeIsReadOnly [](start = 24, length = 20)

seems to only be used once. Consider making it a local function

Reply:
Style of this file seems to be using top level methods. My 2 cents: would maintain the current style and possibly follow up later by making everythngi a local func. 



---
In reply to: [267470431](https://github.com/dotnet/roslyn/pull/34260#discussion_r267470431) [](ancestors = 267470431)

Reply:
Would like to keep uniformity with `InitializeIsExtensionMethod` and similar methods in the file. Not all of these similar-looking methods can really be moved as `GetAttributes` may also call `InitializeIsExtensionMethod`.